validation/src/test/scala/hmda/validation/rules/lar/validity/PurchaserTypeUtils.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
```

Comment:
Making a separate utility for generating bogus data makes sense (especially since it will probably be useful for future edit check tests). I like it.

I haven't thought through where they should live. Want to tell me about your reasoning for putting it here?